src/tests/unit/tests/DetailsView/components/report-export-component.test.tsx

```diff
@@ -82,5 +84,28 @@ describe('ReportExportComponentTest', () => {
 
             expect(wrapper.getElement()).toMatchSnapshot('user input new discription');
         });
+
+        test('clicking export on the dialog should trigger the generateHtml', () => {
+            const wrapper = shallow(<ReportExportComponent {...props} />);
+
+            reportGeneratorMock
+                .setup(rgm => rgm.generateName(props.exportResultsType, props.scanDate, props.pageTitle))
+                .verifiable(Times.once());
```

Comment:
I needed it because I was simulating a `click` on the first button, but now I am directly calling function from `props`; which let me by pass this. So removed the `click` and the `generateName` mock verification.